internal/pkg/config/watcher/watcher.go

```diff
@@ -149,7 +149,7 @@ func mapStringStringToLabelSelector(m map[string]string) string {
 // Get fetches all matching ConfigMaps
 func (c *K8sConfigMapWatcher) Get() (cfgs []*config.InjectionConfig, err error) {
```

Comment:
This should probably be refactored to match `K8sConfigMapWatcher.Watch`'s context parameter in position 1 instead of context.Background(), to keep things tidy.